Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne.py

```diff
@@ -1169,20 +1212,139 @@ def submit_file_to_sandbox(client: Client, args: Dict[str, Any]) -> Union[str, C
         response = result.json()
 
     message = {
-        'message': response.get("message", ""),
-        'code': response.get("code", ""),
-        'task_id': response.get("data", "").get("taskId", ""),
-        'digest': response.get("data", "").get("digest", ""),
+        "message": response.get("message", ""),
+        "code": response.get("code", ""),
+        "task_id": response.get("data", "").get("taskId", ""),
+        "digest": response.get("data", "").get("digest", ""),
+    }
+    results = CommandResults(
+        readable_output=tableToMarkdown(
+            TABLE_SUBMIT_FILE_TO_SANDBOX, message, removeNull=True
+        ),
+        outputs_prefix="VisionOne.Submit_File_to_Sandbox",
+        outputs_key_field="message",
+        outputs=message,
+    )
+    return results
+
+
+def submit_file_entry_to_sandbox(
+    client: Client, args: Dict[str, Any]
+) -> Union[str, CommandResults]:
+    entry = args.get(ENTRY_ID)
+    file_ = demisto.getFilePath(entry)
+    file_name = file_.get("name")
+    file_path = file_.get("path")
+    archive_pass = args.get(ARCHIVE_PASSWORD)
+    document_pass = args.get(DOCUMENT_PASSWORD)
+    try:
+        if not file_path == "test/test.txt":
```

Comment:
Hey @omerKarkKatz Thank you for getting back to me.
1. For the test_data files, I was going over the pre PR integrations checklist (https://xsoar.pan.dev/docs/contributing/checklist#integrations) and point 8 asks to add unit tests data. I had also asked on DFIR community in slack what that would be and someone suggested that I need to add mock json responses for the API.
2. I added contents = b'Hello World' because during linting I kept getting a fail saying the file path could not be found. Looks like getFilePath() from demisto-content/Tests/demistomock/demostomock.py always returns the path 'test/test.txt' and I'm not sure if I should be making a change to this file for testing. 

Please let me know what changes I should make. I appreciate your feedback!